app/dao/services_dao.py

```diff
@@ -306,7 +307,7 @@ def dao_create_service(
         if organisation.letter_branding and not service.letter_branding:
             service.letter_branding = organisation.letter_branding
 
-    elif service.organisation_type in ['nhs_central', 'nhs_local'] or email_address_is_nhs(user.email_address):
+    elif service.organisation_type in NHS_ORGANISATION_TYPES or email_address_is_nhs(user.email_address):
```

Comment:
do we test automatically giving NHS branding somewhere? 🤔

Reply:
Yeah, I added an extra test case here: https://github.com/alphagov/notifications-api/pull/2600/files#diff-2be09a40f09ecb7eb899c53704ae81efR147